treestats_paper.tex

```diff
@@ -252,10 +265,12 @@ \section*{Framework and statistics}
 i.e., $\iw(v)$ may be a vector $(\iw_1(v), \ldots, \iw_m(v))$,
 so that when summarising we have access to more than one aspect of each subtree.
 The requirement that $f(0) = f(\tiw) = 0$ is for consistency,
-so that statistics do not depend on portions of the tree not ancestral to any of the samples.
+so that statistics do not depend on portions of the tree either not ancestral to any of the samples or ancestral to all
+of them.
 % Below, in section XXX, we describe how to efficiently
 % maintain a list of the weights of the subtrees below each node
 % as we iterate through the tree sequence.
+\krt{Is this true?  If so, why isn't the current Tajima's D implentation done in one pass through the tree sequence?}
```

Comment:
I'm not sure what you mean?  There's two reasons Tajima's D is how it is now: one is that we haven't bothered; but the other is that Tajima's D is a ratio of two (additive) statistics of the form we describe here.

Reply:
If the API lets the stat be vector-valued, then I would expect that D should be implemented in a single pass through the tree sequence, where the function fill the first element of the vector with `pi` and the second element with `S`.

Reply:
Yep, exactly. Just haven't done that.